.github/workflows/run_tests.yml

```diff
@@ -47,7 +48,26 @@ jobs:
         export CLARIFAI_USER_ID="$(python scripts/key_for_tests.py --get-userid)"
         export CLARIFAI_PAT="$(python scripts/key_for_tests.py --create-pat)"
         if [ "${{ github.event.pull_request.head.repo.full_name }}" = "${{ github.repository }}" ]; then
-          pytest --cov=.
+          pytest --cov=. --cov-report=xml:coverage/coverage.cobertura.xml
         else
-          pytest -m "not requires_secrets" --cov=.
+          pytest -m "not requires_secrets" --cov=. --cov-report=xml:coverage/coverage.cobertura.xml
         fi
+    - name: Code Coverage Report
+      if: runner.os == 'Linux'
+      uses: irongut/CodeCoverageSummary@v1.3.0
+      with:
+        filename: coverage/**/coverage.cobertura.xml
```

Comment:
Why is there another level in the directory `**` ?

Reply:
It will match any file named coverage.cobertura.xml in the coverage/ directory or any subdirectory under coverage/, since worflows are running in isloated environment and to cover the edge cases.